packages/input-container/src/vaadin-input-container.js

```diff
@@ -19,6 +19,14 @@ export class InputContainer extends ThemableMixin(DirMixin(PolymerElement)) {
           display: flex;
           align-items: center;
           flex: 0 1 auto;
+          border-radius:
+            /* top left, top right, bottom right, bottom left: https://developer.mozilla.org/docs/Web/CSS/border-radius */
```

Comment:
I was thinking about that too - copying CSS logical props is too confusing (especially vertical dimensions).
How about if we use `top-start` and `bottom-end` etc? This would be in line with tooltip position etc.

Reply:
Renamed.

Reply:
I meant that for example `--vaadin-input-container-top-start-radius` will affect the top-left (instead of "start") corner regardless if you have RTL or LTR on.

Reply:
Right, my bad. Then it probably makes sense to use `top-left` for now and update #5268 accordingly.

Reply:
There are also logical props for border radius, such as `border-start-start-radius`, so we could use those instead of the shorthand?

Reply:
Yes, that was my original idea. As mentioned here https://github.com/vaadin/web-components/pull/5251#pullrequestreview-1239270951, we use Chrome 88 for visual tests.

IMO we should first finish #5253 and then update this PR to use logical props.